test/integration/features/profile.feature

```diff
@@ -160,3 +162,28 @@ Feature: Profile
      Then Minishift should have state "Does Not Exist"
      When executing "minishift ip"
      Then exitcode should equal "1"
+
+  Scenario: As user, I cannot create profile when used along with profile <subcommand> --profile name
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift profile list --profile foo" succeeds
+     Then stdout should not contain
+        """
```

Comment:
@praveenkumar Please refer the below link and do the indentation accordingly 

https://github.com/minishift/minishift/blob/master/test/integration/features/profile.feature#L19